src/content/plugins/ignore-plugin.md

```diff
@@ -23,3 +23,5 @@ As of [moment](https://momentjs.com/) 2.18, all locales are bundled together wit
 ```js
 new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/)
 ```
+
```

Comment:
@maggiepint I'm wondering if a better way to put this might be…

**Note:** The Moment team shipped a [new libarary called Luxon](https://moment.github.io/luxon/) in November of 2017. Luxon has an API similar to Moment's, but it is smaller, immutable, and properly modularized. We recommend this library for all projects that require only evergreen browser support.

Reply:
Also: `libarary` => `library` and this should probably be a `T>` (custom blockquote syntax for a tip).